underfs/swift/pom.xml

```diff
@@ -81,7 +51,13 @@
     <dependency>
       <groupId>org.javaswift</groupId>
       <artifactId>joss</artifactId>
-      <version>0.9.8</version>
+      <version>0.9.10</version>
+      <exclusions>
+          <exclusion>
+              <groupId>org.slf4j</groupId>
+              <artifactId>slf4j-api</artifactId>
+          </exclusion>
```

Comment:
The reason I ask is because we explicitly include `slf4j-api` as a top level dependency, so the transitive dependency should not be pulled in. Could you verify if the warnings occur without the exclusion but with the above inclusion?